src/App.js

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
 import './App.css';
+import {MovieList} from './components/MovieList';
+import {MediaPlayer} from './components/MediaPlayer';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+    constructor (props) {
+       super(props)
+       this.state = {
+          movieUrl: ''
+       }
+
+       this.handleClick = this.handleClick.bind(this);
+    }
+
+
```

Comment:
use either one or two line breaks between lines, one is recommended and used everywhere and stick to convention followed.

https://github.com/airbnb/javascript/blob/master/react/README.md#methods